rpqueue/__init__.py

```diff
@@ -869,6 +871,10 @@ def quit_on_signal(signum, frame):
 
 SUCCESS_LOG = None
 
+def set_client(client):
+    from util.feature_flags_v2 import FeatureFlag
```

Comment:
I don't think the rpqueue lib should be context awareness of its dependent (Hermosa in this case) and the associate modules this seems anti-pattern and

Reply:
Should we consider initializing a new LD instance within the task itself if performance isn't an issue?

Reply:
That's something we initially tried (and it did work). But it does seem a bit clunky to me, I don't really like the idea of constantly creating something over and over.

Another idea I was exploring was creating the LDClient much like how we are here, but passing it down the call stack, and into the actual unit of work? Maybe there's some avenue of overriding the `ld_client` in hermosa vs in here?

Reply:
Alright, turns out just calling `__import__.some_function()` within the `_tasks.py` solves our issue of having the Hermosa code within the rpq library.